app/helpers/session_helper.rb

```diff
@@ -1,6 +1,10 @@
 module SessionHelper
   def show_session(question)
-    session[:registration][question] ? session[:registration][question].titleize : nil
+    if question == 'uk_or_overseas' and session[:registration][question] == 'Uk'
```

Comment:
`&&` instead of `and` - I would extract `session[:registration][question]` to a variable as well to DRY this up:

```
some_var = session[:registration][question]
if question == '...' && some_var == 'Uk' ...
```

Also could do with a unit test to cover this